sync/fromDisk/questions.js

```diff
@@ -52,6 +58,8 @@ module.exports.sync = function(courseInfo, questionDB, jobLogger, callback) {
                 external_grading_entrypoint: (q.externalGradingOptions && q.externalGradingOptions.entrypoint),
                 external_grading_timeout: (q.externalGradingOptions && q.externalGradingOptions.timeout),
                 external_grading_enable_networking: (q.externalGradingOptions && q.externalGradingOptions.enableNetworking),
+                thumbnail_filename: filename,
+                filename_location: location,
```

Comment:
Can we make this more concise like the above external grading options?

```suggestion
                thumbnail_filename: (q.thumbnail && q.thumbnail.filename),
                filename_location: (q.thumbnail && q.thumbnail.location),
```